playbook/app/pb_kits/playbook/pb_background/docs/_background_category.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import { Background } from '../..'
+import { Flex } from 'playbook-ui'
```

Comment:
This lin ehas to change to `import { Flex } from '../..'` in order to prevent the webpack issue from happening again

Reply:
Ahh got it, thanks